### PR TITLE
update docs a bit

### PR DESCRIPTION
**Pull request by phoenixAja**
### Summary:
- **What:** added some small documentation on how to use aspencli and updated outdated section on adding new users to staging and prod
- **Ticket:** no ticket
- **Env:** no env

### Checklist:
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: docs/backend/README.md [hunk 1]
```diff
@@ -81,18 +81,22 @@ You will also need to rebuild local running docker containers by running `make l
 If you add a third-party library (directly or indirectly) that does not support [python typing](https://docs.python.org/3/library/typing.html), then you may need to add an entry to [`mypy.ini`](../../mypy.ini) to let mypy know [not to expect type hints for that library](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-type-hints-for-third-party-library).
 
 
-## Adding new users to the staging db:
-* get username / password credentials from aws secrets manager (:
-  * `aws secretsmanager get-secret-value --secret-id aspen-config`
+## Adding new users to the staging/prod db:
+Create a new user to the auth0 covidtracker tenet, take note of auth0 user id
```
Comment (vincent-czi) [inline docs/backend/README.md:85]: Is there a wiki or something elsewhere on how to go through this process?

**Review by vincent-czi — APPROVED** [review]
Looks good!

File: docs/backend/README.md (hunk 1 above)
Comment (jgadling) [reply docs/backend/README.md:85]:
```suggestion
Create a new user to the auth0 covidtracker tenant, take note of auth0 user id
```

Comment (jgadling) [reply docs/backend/README.md:85]: Also these instructions are a bit 🤡 - we need an api endpoint and aspencli command to create users asap!

Comment (phoenixAja) [reply docs/backend/README.md:85]: not really.. it's more of a like show someone new how to do it situation. I can try to add some docs for that tomorrow

Comment (phoenixAja) [reply docs/backend/README.md:85]: in our aspen wiki

Comment (phoenixAja) [reply docs/backend/README.md:85]: i mean.. i was just updating it from something that was _really_ outdated.. we actually do have an endpoint to create new users [here](https://github.com/chanzuckerberg/aspen/blob/36ba8037652998e3e997a21fcc68b5fb87150e2f/src/backend/aspen/app/views/usergroup.py#L72) see the post section, that's where new users get created, I haven't gotten around to testing this endpoint yet with the new aspencli..

Comment (phoenixAja) [reply docs/backend/README.md:85]: https://app.shortcut.com/genepi/story/167164/add-aspen-cli-command-to-create-new-user

**Pull request merged**